src/docfx/lib/StringUtility.cs

```diff
@@ -37,6 +37,10 @@ public static string UpperCaseFirstChar(string value)
 
         public static string Join<T>(IEnumerable<T> source, int take = 5)
         {
+            if (!source.Any())
```

Comment:
nit: test string result from line 45 to avoid enumerating `source` twice. Because this is `IEnumerable<T>`, it could be slow to enumerate.

Reply:
`source.Any()` would only take 1 iteration, but sure we can save this 1 iteration to reuse the result from line 45.